src/main/java/org/gridsuite/merge/orchestrator/server/NetworkConversionService.java

```diff
@@ -45,21 +68,121 @@ public NetworkConversionService(RestTemplate restTemplate) {
         this.networkConversionServerRest = restTemplate;
     }
 
-    public ExportNetworkInfos exportMerge(List<UUID> networksIds, String format, String baseFileName) {
-        UriComponentsBuilder uriBuilder = UriComponentsBuilder.fromPath(DELIMITER + NETWORK_CONVERSION_API_VERSION + "/networks/{networkUuid}/export/{format}");
+    public void setCaseFetcherService(CaseFetcherService caseFetcherService) {
```

Comment:
we don't have setters for the other dependencies, why add this one ?

Reply:
Done